### ISSUE TITLE
error running `setup:all`

### ISSUE THREAD

**Issue by samnissen** [samnissen/polariscope_2#25]
```
-----> Setup the DB
$
    echo "-----> Create SQL query"
    DATABASE=polariscope_two
    Q1="CREATE DATABASE IF NOT EXISTS $DATABASE;"
    Q2="GRANT USAGE ON polariscope_two.* TO $PSAAPPMYSQLUSERNAME@localhost IDENTIFIED BY '$PSAAPPMYSQLPASSWORD';"
    Q3="GRANT ALL PRIVILEGES ON polariscope_two.* TO $PSAAPPMYSQLUSERNAME@localhost;"
    Q4="FLUSH PRIVILEGES;"
    SQL="${Q1}${Q2}${Q3}${Q4}"
    echo "-----> Execute SQL query to create DB and user"
    echo "-----> Enter MySQL root password on prompt below"
    echo \$\ mysql\ -uroot\ -p\ -e\ \"\$SQL\" &&
mysql -uroot -p -e "$SQL"
    echo "-----> Done"

-----> Create SQL query
-----> Execute SQL query to create DB and user
-----> Enter MySQL root password on prompt below
$ mysql -uroot -p -e "$SQL"
Enter password:
ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '@localhost IDENTIFIED BY ''' at line 1
```

**Comment by jhulme** [comment]
Issue here is likely that the environment variables are not available to the remote shell. Need to make sure that the entries are added to ~/.bashrc

**Comment by samnissen** [comment]
Fixed by #35
